tests/test_oneof.py

```diff
@@ -0,0 +1,72 @@
+import unittest
+
+TestOneof = None
+SubMsg = None
+
+
+class TestOneofTest(unittest.TestCase):
```

Comment:
Thank you for the tests!!!

Reply:
Welcome! I wouldn't have deployed this without them. I would say there's room for improving the docs surrounding the tests but ultimately I got them set up.